MovieApp/MovieApp/MovieCell.swift

```diff
@@ -25,10 +26,10 @@ class MovieCell: UICollectionViewCell {
         fatalError("init(coder:) has not been implemented")
     }
     
-    func populate(withMovie movie: Movie) {
+    func populate(withMovie movie: MovieViewModel) {
         movieTitle.text = movie.title
         movieDescription.text = movie.overview
-        movieImageView.image = UIImage(named: movie.posterPath)
+        movieImageView.kf.setImage(with: URL(string: "https://image.tmdb.org/t/p/w185\(movie.posterPath)"))
```

Comment:
It's not recommended to have a hardcoded URL in the `UICollectionViewCell`, not even in the view model.
For the current architecture, I'd suggest doing this logic in presenter while creating a view model, so you'd just write:
```swift
movieImageView.kf.setImage(with: movie.posterPath)
```